### PR TITLE
changed picture

### PR DESCRIPTION
**Pull request by mbarger14**
fixed the picture

### CODE REVIEW

**Review by michaelwnau — COMMENTED** [review]
Check the formatting for the frontmatter. They way it is in the commit will break the way `tags` and `categories` work. 
Use this format:
---
author: alexis
categories:
- announcements
- ZCG
layout: post
post_image: /assets/images/logos/logo_3.webp
tags:
- news
title: Introducing zCore Group
---

**Comment by michaelwnau** [comment]
Screenshot 2023-06-30 122143

**Review by michaelwnau — APPROVED** [review]
Reviewed and approved changes to title - resolved merge conflict.

**Review by michaelwnau — DISMISSED** [review]
Reviewed and approved changes to title - resolved merge conflict.

File: _posts/2023/05/2023-05-19-building-an-effective-pipeline.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-
+---
+author: mackenzie
+categories:
+- announcements
+layout: post
+post_image: "/assets/images/blog/pipeline.jpeg"
+tags:
+- business
+title: " Mastering the Art of Building an Effective Pipeline with Federal Contracts"
```
Comment (bastosmichael) [inline _posts/2023/05/2023-05-19-building-an-effective-pipeline.md:9]: remove the space between " and Mastering

**Review by michaelwnau — APPROVED** [review]
confirmed md styling restored in headlines

**Review by michaelwnau — APPROVED** [review]
confirmed md styling restored in headlines

**Review by michaelwnau — APPROVED** [review]
confirmed md styling restored in headlines

**Pull request merged**